Source/MgsLib/LibDG.cpp

```diff
@@ -2667,41 +2667,28 @@ MGS_FUNC_IMPLEX(0x407122, LibGV_407122, LIBDG_IMPL);
 
 void CC OrderingTableAdd_4034C6(int pPrimDataStart, int count, int size)
 {
-    MGS_FORCE_ENOUGH_SPACE_FOR_A_DETOUR;
-
-    /*
-    DWORD* dword_991E40_1_ot_ptr = (DWORD*)gMatrix_dword_991E40[1];
-    const int dword_991E40_2_field_2E_w_or_h = gMatrix_dword_991E40[2];
-    if (count - 1 >= 0)
+    DWORD* dword_991E40_1_ot_ptr = (DWORD*)((DWORD*)gScratchPadMemory_991E40.field_0_raw.field_0)[1];

```

Comment:
the scratch pad memory here is `ScratchPad_Allocs* pScratch = (ScratchPad_Allocs*)&gScratchPadMemory_991E40; // TODO: Add to union` 

Probably better to use it as that type.